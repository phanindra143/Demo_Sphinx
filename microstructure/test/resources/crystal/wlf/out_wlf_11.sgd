*if,ifprop,eq,2,then
  !c_ij(1,1)=     0.0000
  !c_ij(1,2)=     0.0000
  !c_ij(2,2)=     0.0000
  !c_ij(1,3)=     0.0000
  !c_ij(2,3)=     0.0000
  !c_ij(3,3)=     0.0000
  !c_ij(1,4)=     0.0000
  !c_ij(2,4)=     0.0000
  !c_ij(3,4)=     0.0000
  !c_ij(4,4)=     0.0000
  !c_ij(1,5)=     0.0000
  !c_ij(2,5)=     0.0000
  !c_ij(3,5)=     0.0000
  !c_ij(4,5)=     0.0000
  !c_ij(5,5)=     0.0000
  !c_ij(1,6)=     0.0000
  !c_ij(2,6)=     0.0000
  !c_ij(3,6)=     0.0000
  !c_ij(4,6)=     0.0000
  !c_ij(5,6)=     0.0000
  !c_ij(6,6)=     0.0000
*endif
*if,ifprop,eq,1,then
  !k_ij(1,1)=   118.8597
  !k_ij(1,2)=     0.4588
  !k_ij(2,2)=   118.7356
  !k_ij(1,3)=    -0.4367
  !k_ij(2,3)=     0.0017
  !k_ij(3,3)=   118.7263
MP,Kxx,1,  118.85966
MP,Kyy,1,  118.73560
MP,Kzz,1,  118.72626
MP,Kxx,2,  118.73560
MP,Kyy,2,  118.85966
MP,Kzz,2,  118.72626
MP,Kxx,3,  118.73560
MP,Kyy,3,  118.85966
MP,Kzz,3,  118.72626
MP,Kxx,4,  118.85966
MP,Kyy,4,  118.73560
MP,Kzz,4,  118.72626
MP,Kxx,5,  118.85966
MP,Kyy,5,  118.73560
MP,Kzz,5,  118.72626
MP,Kxx,6,  118.73560
MP,Kyy,6,  118.85966
MP,Kzz,6,  118.72626
MP,Kxx,7,  118.73560
MP,Kyy,7,  118.85966
MP,Kzz,7,  118.72626
MP,Kxx,8,  118.85966
MP,Kyy,8,  118.73560
MP,Kzz,8,  118.72626
MP,Kxx,9,  118.72626
MP,Kyy,9,  118.85966
MP,Kzz,9,  118.73560
MP,Kxx,10,  118.85966
MP,Kyy,10,  118.72626
MP,Kzz,10,  118.73560
MP,Kxx,11,  118.85966
MP,Kyy,11,  118.72626
MP,Kzz,11,  118.73560
MP,Kxx,12,  118.72626
MP,Kyy,12,  118.85966
MP,Kzz,12,  118.73560
MP,Kxx,13,  118.72626
MP,Kyy,13,  118.85966
MP,Kzz,13,  118.73560
MP,Kxx,14,  118.85966
MP,Kyy,14,  118.72626
MP,Kzz,14,  118.73560
MP,Kxx,15,  118.85966
MP,Kyy,15,  118.72626
MP,Kzz,15,  118.73560
MP,Kxx,16,  118.72626
MP,Kyy,16,  118.85966
MP,Kzz,16,  118.73560
MP,Kxx,17,  118.73560
MP,Kyy,17,  118.72626
MP,Kzz,17,  118.85966
MP,Kxx,18,  118.72626
MP,Kyy,18,  118.73560
MP,Kzz,18,  118.85966
MP,Kxx,19,  118.72626
MP,Kyy,19,  118.73560
MP,Kzz,19,  118.85966
MP,Kxx,20,  118.73560
MP,Kyy,20,  118.72626
MP,Kzz,20,  118.85966
MP,Kxx,21,  118.73560
MP,Kyy,21,  118.72626
MP,Kzz,21,  118.85966
MP,Kxx,22,  118.72626
MP,Kyy,22,  118.73560
MP,Kzz,22,  118.85966
MP,Kxx,23,  118.72626
MP,Kyy,23,  118.73560
MP,Kzz,23,  118.85966
MP,Kxx,24,  118.73560
MP,Kyy,24,  118.72626
MP,Kzz,24,  118.85966
*endif
