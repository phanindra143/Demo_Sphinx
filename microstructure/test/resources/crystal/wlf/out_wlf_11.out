            ifx                 q_x                 q_y                 q_z     
  /% source: R:/166_320/Ablage_Simulationen/Modellbildung_Simulation_CMC-SiC_[Vorlauf+WiMi+MaVo]/04_04_2017_Modellierung_Matrix_mikro_ditt_[WiMi]/2_Waermeleitfähigket_SiC-Si\WLF-50SiC-Poren\2
             3.00000      -1857163.33287           233.15775          9744.68316
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             3.00000         15625.00000           -63.74936           -25.62363
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             2.00000            36.79558      -1855216.66046          8253.61172
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             2.00000           -59.46543         15625.00000           -70.35879
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             1.00000          9897.87820         12483.59964      -1855108.65970
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             1.00000           -24.44114          -104.88770         15625.00000
  /%
